exec/exec.go

```diff
@@ -12,12 +12,14 @@ import (
 // dcos-go/exec is a os/exec wrapper. It implements io.Reader and can be used to read both STDOUT and STDERR.
 //
 // Usage:
-// ce := exec.Run("bash", []string{"infinite.sh"}, exec.Timeout(3 * time.Second))
+// ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+// defer cancel()
 //
+// ce, _ := exec.Run(ctx, "bash", []string{"infinite.sh"})
```

Comment:
Indeed it feels wrong to ignore the error, I think I had left it because that function implementation never returns any error at the moment. Do you mean this might be here for later implementation?